build/mips-simd-check.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#**********************************************************************************
+#    This script is using in build/arch.mk for mips to detect the simd instructions:
+#    mmi, msa (maybe more in the future).
+#
+#   --usage:
+#             ./mips-simd-check.sh $(CC) mmi
+#         or  ./mips-simd-check.sh $(CC) msa
+#
+# date:  10/17/2019 Created
+#**********************************************************************************
+
+TMPC=$(mktemp test.XXXXXX.c)
+TMPO=$(mktemp test.XXXXXX.o)
+if [ $2 == "mmi" ]
+then
+   echo "void main(void){ __asm__ volatile(\"punpcklhw \$f0, \$f0, \$f0\"); }" > $TMPC
+   $1 -Wa,-mloongson-mmi $TMPC -o $TMPO &> /dev/null
```

Comment:
the same as $TMPC, $TMPO should be deleted

Reply:
yes, the last line of mips-simd-check.sh had rm -f $TMPC, $TMPO.

Reply:
sorry for missing out.